packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -82,6 +88,18 @@ export default function usePatternDetails( postType, postId ) {
 					? __( 'Not synced' )
 					: __( 'Fully synced' ),
 		} );
+		if ( record.wp_pattern_category?.length > 0 ) {
+			const categories = record.wp_pattern_category
+				.filter( ( category ) => patternCategories.get( category ) )
+				.map( ( category ) => patternCategories.get( category ).label );
+			details.push( {
+				label: __( 'Categories' ),
+				value:
+					categories.length > 0
+						? categories.join( ', ' )
+						: __( 'Uncategorized' ),
+			} );
+		}
```

Comment:
With the initial check for `.length > 0` we'll not reach the `Uncategorized` label here. 

I think we should be adding whether a pattern is currently uncategorized to the details screen so best if we can fix this up.

Reply:
Ah, with that `uncategorized` I was thinking of instances where the category id may be assigned but may  not match the existing categories, eg. if user assigned a category, but then deleted it from category management page, but it looks like WP nicely tidies that up, so I have [updated to correctly assign uncategorized if there are no category ids](https://github.com/WordPress/gutenberg/pull/53835/commits/11ae9f6c319dcf1aa041e12d0df6548d81653a8d).